packages/astro/src/assets/fonts/config.ts

```diff
@@ -2,7 +2,8 @@ import { z } from 'zod';
 import { LOCAL_PROVIDER_NAME } from './constants.js';
 
 const weightSchema = z.union([z.string(), z.number()]);
-const styleSchema = z.enum(['normal', 'italic', 'oblique']);
+export const styleSchema = z.enum(['normal', 'italic', 'oblique']);
+const unicodeRangeSchema = z.array(z.string());
```

Comment:
Would this make sense to keep this schema as nonempty? Does an empty range make sense?

Reply:
```suggestion
const unicodeRangeSchema = z.array(z.string()).nonempty();
```